manifests/config.pp

```diff
@@ -3,10 +3,11 @@
 class prometheus::config (
   $global_config,
   $rule_files,
-  $scrape_configs,
+  Array[Hash] $scrape_configs,
```

Comment:
Datatypes \o/

Reply:
Do you know the structure of the Hash? Could that be enforced as well within the datatype?

Reply:
how about we just don't try to fix that in this specific pull request? there wasn't a type before either...
```suggestion
  $scrape_configs,
```

:P